MovieTime/MovieTime/Core/MTLoadingProtocol.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  MTLoadingProtocol.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 12/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+protocol MTLoadingProtocol: class {
```

Comment:
how do you envision this protocol being reused in a future version of this app?

Reply:
It is possible that the control of Loading will be done by the network layer and / or other classes, using a protocol that can be extended (*UIRefreshControl, UIActivityIndicatorView, UIView, UIViewController, etc.*) making easy to test the methods and helps the maintenance (if I want to change a Loading over all `View`, to a contextualized loading `ActiviyIndicator`).